app/User.php

```diff
@@ -441,6 +449,26 @@ public function isDripSubscriber()
       return ! is_null($this->drip_subscriber_id);
     }
 
+    public function isRepairDirectoryNone()
+    {
+        return $this->repairdir_role == Role::REPAIR_DIRECTORY_NONE;
```

Comment:
This looks like it is expecting repairdir_role of 0 to be set in the DB?  What would happen if it was null?

Reply:
Double =, so 0 == NULL.  

It's not pleasant, but I don't want to mess with select2, really.